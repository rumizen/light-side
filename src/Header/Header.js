import React from 'react';
import './Header.scss';
import FavoritesBar from '../FavoritesBar/FavoritesBar.js';
import NavBar from "../NavBar/NavBar.js";

const Header = ({ faveCount }) => {
  return (
    <header className="header">
      <h1>Dark Side</h1>
      <div className="nav-button-style">
        <FavoritesBar faveCount={faveCount}/>
        <NavBar />
      </div>
    </header>
  );
};

export default Header;