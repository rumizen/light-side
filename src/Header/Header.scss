.header {
  display: flex;
  justify-content: space-around;
  align-items: center;
}

.nav-button-style {
  display: flex;
}

button {
  height: 62px;
  margin: 0px 15px;
  width: 90px;
}

h1 {
  font-size: 72px;
  text-shadow: 3px 3px 3px white
}