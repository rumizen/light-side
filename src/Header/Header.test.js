import React from "react";
import { shallow, mount } from "enzyme";
import Header from "./Header";

describe("Header", () => {
  it("should match the Header snapshot", () => {
    const wrapper = shallow(
    <Header faveCount={5} 
    />);

    expect(wrapper).toMatchSnapshot();
  });
});
