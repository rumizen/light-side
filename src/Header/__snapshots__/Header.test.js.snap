// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the Header snapshot 1`] = `ShallowWrapper {}`;
